GeneticAlgorithm.py

```diff
@@ -689,6 +691,30 @@ def set_meetingTime(self, meetingTime): self._meetingTime = meetingTime
         get_labs
     Setters:
 '''
+
+
+#class GettersModel(models.Model):
+    #GATA = models.CharField()
+    #Course = models.TextField()
+    #Labs = models.TextField()
+    #Assignment = models.TextField()
+    #SemesterYear = models.CharField()
+    #Schedules = models.TextField()
+
+class Getters():
```

Comment:
Is this the best way of getting data from the database? We should be making an API call and selecting data given a specific Semester and Year.